tests/test_endpoints.py

```diff
@@ -21,7 +21,8 @@
 from pathlib import Path
 
 from charmhelpers.core import unitdata
-from charms.reactive import Endpoint, set_flag, is_flag_set, clear_flag
+from charms.reactive import Endpoint
+from charms.reactive import set_flag, is_flag_set, clear_flag, register_trigger
```

Comment:
A bit weird here. Normally you switch to multi-line imports if the line grows too long.